concepts/05 Widgets/Scheduler/010 Overview.md

```diff
@@ -79,6 +79,79 @@ The following code adds the **Scheduler** widget to your page. The simplest conf
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxScheduler>
+            :data-source="dataSource"
+            :current-date="currentDate"
+        </DxScheduler>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxScheduler, {
+        DxScheduler
+    } from 'devextreme-vue/scheduler';
+
+    export default {
+        components: {
+            DxScheduler
+        },
+        data() {
+            return {
+                currentDate: new Date(2016, 4, 25),
+                dataSource: [{
+                    text: "Website Re-Design Plan",
+                    startDate: new Date(2016, 4, 25, 9, 30),
+                    endDate: new Date(2016, 4, 25, 11, 30)
+                }, {
+                    text: "Book Flights to San Fran for Sales Trip",
+                    startDate: new Date(2016, 4, 25, 12, 0),
+                    endDate: new Date(2016, 4, 25, 13, 0)
+                }, 
+                // ...
+                ]
+            }
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import {Scheduler} from 'devextreme-react/scheduler';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <Scheduler>
+                    dataSource={[{
```

Comment:
Please declare the data source as a constant or component property to avoid unnecessary re-rendering.

Reply:
Please address this issue.